tests/features/content_types/article.feature

```diff
@@ -20,3 +20,20 @@ Feature: Article content type
     And I should see the heading "EC decides tax advantages for Fiat are illegal"
     And I should see the text "Commissioner states tax rulings are not in line with state aid rules."
     And I should see the text "Published by Anonymous"
+
+  @api @europa_theme
```

Comment:
Ok, for the 2 files but the 3rd one does not generate a duplication of the scenarios?